### ISSUE TITLE
[Docker Desktop] Application Designer in Community 

### ISSUE THREAD

**Issue by nebuk89** [docker/roadmap#70]
**Tell us about your request**
Release Docker Desktop application designer as part of Desktop community to give users a UI to build new projects from pre-existing templates. To learn more about app designer: https://docs.docker.com/ee/desktop/app-designer/

**Which service(s) is this request for?**
Docker Desktop 

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
Developers are creating more new projects than ever before, getting started with a new project or a new technology stack takes time to bring all these pieces together. 

**Are you currently working around the issue?**
N/A

**Additional context**
N/A

**Comment by xinity** [comment]
YES! this feature would be really really really useful to ease new developers on-boarding.
another thing would be a way to "share" templates made by the community :)
